src/App/App.tsx

```diff
@@ -1,30 +1,25 @@
-import { useEffect, useState, useCallback } from 'react';
-import AddItem from 'AddItem/AddItem';
+import { useState, useCallback } from 'react';
+import { AddItem } from 'AddItem/AddItem';
```

Comment:
seems to be changed to:
```suggestion
import AddItem from 'AddItem/AddItem';
```